data/themes/default/style.css

```diff
@@ -620,15 +620,17 @@ TrackContentObjectView   {
 	qproperty-selectedColor: #006B65;
 	qproperty-BBPatternBackground: #373d48;
 	qproperty-textColor: #fff;
+	qproperty-textBackgroundColor: rgba(0, 0, 0, 75);
 	qproperty-textShadowColor: rgb(0,0,0,200);
```

Comment:
I still kinda like it with the shadow now that I look at it, makes it more readable imo